src/CodeGen_LLVM.cpp

```diff
@@ -904,6 +904,8 @@ llvm::Type *CodeGen_LLVM::llvm_type_of(Type t) {
     return Internal::llvm_type_of(context, t);
 }
 
+llvm::TargetMachine *get_target_machine(const llvm::Module &module);
```

Comment:
Oh, I see, that would move a bunch of code. Any opinion @dsharlet? I think
you wrote this originally.

On Thu, Apr 28, 2016 at 1:19 PM, Alina Sbirlea <notifications@github.com>
wrote:

> In src/CodeGen_LLVM.cpp
> <https://github.com/halide/Halide/pull/1170#discussion_r61495660>:
>
> > @@ -904,6 +904,8 @@ llvm::Type *CodeGen_LLVM::llvm_type_of(Type t) {
> >      return Internal::llvm_type_of(context, t);
> >  }
> >
> > +llvm::TargetMachine *get_target_machine(const llvm::Module &module);
>
> Sure. Do you suggest moving get_target_options and the two get_md_* as
> well?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/halide/Halide/pull/1170/files/8d9f16fd834fb6d8d490e8452b4edd4f07462517#r61495660>
>